libs/wire-api/test/golden/testObject_WithStatus_team_18.json

```diff
@@ -1,6 +1,6 @@
 {
     "config": {
-        "verificationTimeout": 1676377048
+        "verificationExpiration": 1676377048
```

Comment:
Where does this come from? :thinking:

Reply:
This was related to earlier commits in the PR.